nifi-commons/nifi-site-to-site-client/src/main/java/org/apache/nifi/remote/client/SiteToSiteClient.java

```diff
@@ -112,7 +107,7 @@
      * @throws IOException ioe
      * @throws org.apache.nifi.remote.exception.UnknownPortException upe
```

Comment:
Javadoc should match new throws

Reply:
This was an automatic IDE cleanup. Will update Javadoc.